### DISCUSSION TITLE
Export power limit toggle

### DISCUSSION THREAD

**Discussion by pajadvor** [mletenay/home-assistant-goodwe-inverter#171] [Q&A] [answered]
Is it possible somehow programatically to toggle between power limit enabled and disabled from HA?
The scenario for me is this, i'm not able to export to grid, so i have this switch enabled. 
If enabled the inverter does not try to compensate unless around 300w of load is present. During night it taking around 270w continually from grid. If i disable export power limitation during night, it howers around zero and do not take electricity from grid.
I'd do an automation to turn this on/off depending on PV generation. 
If export power limit is enabled in PV Master, there is no difference in consumption from grid during night, i can set it to zero or 10000w, the approx. 300w threshold is there no matter what.

**Comment by mletenay** [comment, marked as answer]
Yes, it is. See https://github.com/mletenay/home-assistant-goodwe-inverter/issues/167#issuecomment-1462901516

**Reply by pajadvor** [reply]
Thank you for your promt reply, and great work on this plugin/extension.
